src/3d/castletransform_physics.inc

```diff
@@ -193,7 +208,17 @@
       It would free the rigid body instance, which will crash the physics engine for now.
       Instead, you can set @link(TCastleTransform.Exists) to @false
       (you can free it later, e.g. during @link(TCastleTransform.OnUpdate)). }
-    property OnCollisionExit: TOnCollision read FOnCollisionExit write FOnCollisionExit;
+    property OnCollisionExit: TOnCollision read FOnCollisionExit write SetOnCollisionExit;
+
+    { Occurs when TRigidBody still colliding with another TRigidBody.
```

Comment:
"still colliding" -> "still collides" (in English you should say "X still collides with Y" or "X is still colliding with Y")

Reply:
Sorry, I'm still learning English ;)